lms/static/js/search/SearchApp.js

```diff
@@ -1,18 +1,20 @@
-
-var edx = edx || {};
-
-(function(Backbone) {
+RequireJS.define([
```

Comment:
I get an error saying define is undefined. I think it gets undefined in RequireJS-namespace-undefine.js .